core/config.go

```diff
@@ -1,158 +1,33 @@
-/*
```

Comment:
Why are these copyright messages removed?

Reply:
Oops, renamed file than made a new one with the same name. Added them.